src/main/java/com/hivemq/extensions/client/ClientContextImpl.java

```diff
@@ -61,22 +63,30 @@ public void addInterceptor(@NotNull final Interceptor interceptor) {
         }
     }
 
-    public void addPublishInboundInterceptor(@NotNull final PublishInboundInterceptor interceptor) {
+    public void addPingRequestInboundInterceptor(final @NotNull PingRequestInboundInterceptor interceptor) {
```

Comment:
follow the final and NotNull sequence convention of this file please.

Reply:
make sure that every method in this file is in the order that it was before. Otherwise we could encounter ugly merge conflicts later down the road.